### ISSUE TITLE
"Fixed Asset Register" Report Error

### ISSUE THREAD

**Issue by AlaaBadry** [frappe/erpnext#33931]
### Information about bug

Can't open "Fixed Asset Register" Report

### Module

assets

### Version

"erpnext": "14.14.0",
"frappe": "14.24.0"

### Installation method

FrappeCloud

### Relevant log output / Stack trace / Full Error Message.

```shell
### App Versions

{
	"erpnext": "14.14.0",
	"frappe": "14.24.0"
}
```
### Route
```

```
### Trackeback
```
Traceback (most recent call last):
  File "apps/frappe/frappe/app.py", line 69, in application
    response = frappe.api.handle()
  File "apps/frappe/frappe/api.py", line 54, in handle
    return frappe.handler.handle()
  File "apps/frappe/frappe/handler.py", line 45, in handle
    data = execute_cmd(cmd)
  File "apps/frappe/frappe/handler.py", line 83, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "apps/frappe/frappe/__init__.py", line 1609, in call
    return fn(*args, **newargs)
  File "apps/frappe/frappe/__init__.py", line 789, in wrapper_fn
    retval = fn(*args, **get_newargs(fn, kwargs))
  File "apps/frappe/frappe/desk/query_report.py", line 251, in run
    result = generate_report_result(report, filters, user, custom_columns, is_tree, parent_field)
  File "apps/frappe/frappe/__init__.py", line 789, in wrapper_fn
    retval = fn(*args, **get_newargs(fn, kwargs))
  File "apps/frappe/frappe/desk/query_report.py", line 87, in generate_report_result
    res = get_report_result(report, filters) or []
  File "apps/frappe/frappe/desk/query_report.py", line 68, in get_report_result
    res = report.execute_script_report(filters)
  File "apps/frappe/frappe/core/doctype/report/report.py", line 142, in execute_script_report
    res = self.execute_module(filters)
  File "apps/frappe/frappe/core/doctype/report/report.py", line 159, in execute_module
    return frappe.get_attr(method_name)(frappe._dict(filters))
  File "apps/erpnext/erpnext/assets/report/fixed_asset_register/fixed_asset_register.py", line 21, in execute
    prepare_chart_data(data, filters)
  File "apps/erpnext/erpnext/assets/report/fixed_asset_register/fixed_asset_register.py", line 163, in prepare_chart_data
    labels_values_map[belongs_to_month].asset_value += d.get("asset_value")
TypeError: unsupported operand type(s) for +=: 'int' and 'NoneType'

```
### Request Data
```
{
	"type": "POST",
	"args": {
		"report_name": "Fixed Asset Register",
		"filters": "{\"status\":\"In Location\",\"filter_based_on\":\"Fiscal Year\",\"period_start_date\":\"2020-04-01\",\"period_end_date\":\"2021-03-31\",\"date_based_on\":\"Purchase Date\",\"group_by\":\"--Select a group--\",\"company\":\"????\",\"from_fiscal_year\":\"2021\",\"to_fiscal_year\":\"2021\",\"from_date\":\"2022-02-02\",\"to_date\":\"2023-02-02\"}",
		"ignore_prepared_report": 1
	},
	"headers": {},
	"error_handlers": {},
	"url": "/api/method/frappe.desk.query_report.run"
}
```
### Response Data
```
{
	"exception": "TypeError: unsupported operand type(s) for +=: 'int' and 'NoneType'"
}
```
```

**Comment by anandbaburajan** [comment]
@AlaaBadry sorry for the inconvenience, you just need to upgrade to the latest release.

**Comment by barredterra** [comment]
Resolved with #33788
